activeadmin_addons.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.summary     = "Set of addons to help with the activeadmin ui"
   s.description = "Set of addons to help with the activeadmin ui"
   s.license     = "MIT"
+  s.required_ruby_version = '>= 2.7.0'
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>required_ruby_version (2.7, declared in activeadmin_addons.gemspec) and TargetRubyVersion (2.5, which may be specified in .rubocop.yml) should be equal.